utils/vcsdata.go

```diff
@@ -0,0 +1,147 @@
+package utils
+
+import (
+	"bufio"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"sync/atomic"
+
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+)
+
+type (
+	VcsDetails struct {
+		vcsRootDirectory sync.Map //key: path to dot git, value: vcsData
+		vcsDirectory     sync.Map //key: path to folder, value: index of vcsRootDirectory
+		vcsDirectorySize *int32   // Size of vcs folders entries
+	}
+	vcsData struct {
+		url      string
+		revision string
+	}
+)
+
```

Comment:
We use in this file three different terms to describe the git details. We use either details, data and info.
Let's choose one and remain consistent.

Reply:
vcsDetails.